### ISSUE TITLE
1.20.6 bug, does not work

### ISSUE THREAD

**Issue by jurohn** [TechnicallyCoded/Inventory-Rollback-Plus#152]
```
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Enabling InventoryRollbackPlus v1.6.15
02.05 11:55:09 [Server] ERROR Error occurred while enabling InventoryRollbackPlus v1.6.15 (Is it up to date?)
02.05 11:55:09 [Server] INFO java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
02.05 11:55:09 [Server] INFO at InventoryRollbackPlus-1.6.15.jar/com.nuclyon.technicallycoded.inventoryrollback.InventoryRollbackPlus.onEnable(InventoryRollbackPlus.java:54) ~[InventoryRollbackPlus-1.6.15.jar:?]
02.05 11:55:09 [Server] INFO at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
02.05 11:55:09 [Server] INFO at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:519) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
02.05 11:55:09 [Server] INFO at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:639) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:551) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:682) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:441) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:324) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1140) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:326) ~[paper-1.20.6.jar:git-Paper-35]
02.05 11:55:09 [Server] INFO at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Disabling InventoryRollbackPlus v1.6.15
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Setting shutdown state
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Saving player inventories...
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Done saving player inventories!
02.05 11:55:09 [Server] INFO [InventoryRollbackPlus] Plugin is disabled!
```

**Comment by TechnicallyCoded** [comment]
Ah this is a paper bug since they decided to delete version checking the way most devs used to do it. Will fix today.

**Comment by TechnicallyCoded** [comment]
Fixed in commit a3daee2751b4ea7df2fc84082e2ffe3648fdcfbf

**Comment by TechnicallyCoded** [comment]
Resolved as of https://www.spigotmc.org/resources/inventory-rollback-plus-1-8-1-20-x.85811/update?update=542923
